.github/workflows/web_tests.yml

```diff
@@ -0,0 +1,174 @@
+name: Web Integration Tests
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+  workflow_dispatch:
+
+jobs:
+  web_tests:
+    name: Web Tests (WebAssembly)
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    
+    strategy:
+      matrix:
+        flutter_version: [3.19.0, 3.24.0]
```

Comment:
3.29.3 is the currently version supported in our apps, so we can increase the version here

Reply:
Done!